mmdet/datasets/custom.py

```diff
@@ -322,3 +322,34 @@ def evaluate(self,
                 for i, num in enumerate(proposal_nums):
                     eval_results[f'AR@{num}'] = ar[i]
         return eval_results
+
+    def __repr__(self):
+        """Print the number of instance number."""
+        dataset_type = 'Test' if self.test_mode else 'Train'
+        result = f'\n{dataset_type} dataset ' \
+                 f'with number of images {len(self)}, ' \
+                 f'and instance counts: \n'
+        if self.CLASSES is None:
+            return result
+        instance_count = np.zeros(len(self.CLASSES) + 1).astype(int)
+        # count the instance number in each image
+        for idx in range(len(self)):
+            label = self.get_cat_ids(idx)
```

Comment:
Better reload this function in `CocoDataset`

Reply:
Reloaded